website/src/components/Messages/AdminMessageTable.tsx

```diff
@@ -42,45 +49,87 @@ const DateDiff = ({ children }: { children: string | Date | number }) => {
   );
 };
 
-export const AdminMessageTable = (props: StrictOmit<DataTableProps<Message>, "columns">) => {
+export const AdminMessageTable = ({ userId, includeUser }: { userId?: string; includeUser?: boolean }) => {
   const [deleteMessageId, setDeleteMessageId] = useState<string | null>(null);
+  const [activeMessageId, setActiveMessageId] = useState<string>();
+  const { pagination, toNextPage, toPreviousPage } = useCursorPagination();
+  const {
+    data: res,
+    isLoading,
+    error,
+    mutate,
```

Comment:
would be great to rename `mutate` to something that reflects the actual behavior.